src/modules/universe/RadixNodeDiscoveryHardcoded.ts

```diff
@@ -12,16 +12,22 @@ export default class RadixNodeDiscoveryHardcoded implements RadixNodeDiscovery {
 
     private readonly httpProtocol: string
     private readonly wsProtocol: string
+    private readonly bootstrapNodes: string[]
     
     /**
      * Creates an instance of radix node discovery from seed.
-     * @param bootstrapNode Full address to the rpc endpoint of a Radix node in the universe
+     * @param bootstrapNodes List of full addresses to the rpc endpoints of Radix nodes in the universe
      * @param ssl Whether to use SSL encryption for communication wiht the nodes
      */
     constructor(
-        readonly bootstrapNodes: string[],
```

Comment:
What's the reasoning behind this change?

Reply:
I reverted this